src/pages/font.module.scss

```diff
@@ -0,0 +1,63 @@
+@use "@/styles/designSystem.scss" as var;
+
+
+.test{
```

Comment:
So when out team member needs to use the text style you have defined here, they also need to wrap the text style classes inside a .text container?

Reply:
yes